.github/workflows/release-check.yml

```diff
@@ -16,4 +16,4 @@ concurrency:
 
 jobs:
   release-check:
-    uses: marcopolo/unified-github-workflows/.github/workflows/release-check.yml@e66cb9667a2e1148efda4591e29c56258eaf385b
+    uses: ipdxco/unified-github-workflows/.github/workflows/release-check.yml@v1.0
```

Comment:
```suggestion
    uses: ipdxco/unified-github-workflows/.github/workflows/release-check.yml@v1.0.17
```
use latest?

Reply:
I think it automatically takes the latest patch with just 1.0